client/src/common/components/ArchivedMessage.jsx

```diff
@@ -55,7 +55,11 @@ class ArchivedMessage extends PureComponent {
               />
             ) : (
               <FormattedMessage
```

Comment:
Please split out the common message and the additional content. Please try this:

```javascript
const successMessage = (
  <>
    <FormattedMessage id="common.archived-message.was-archived" values={{ name, item }} />
    {!isOwner&& <FormattedMessage id="common.archived-message.was-archived-member" values={{ name, item }} />}
  </>
);
```